core/src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -545,6 +547,7 @@ class CassandraReadJournal(system: ExtendedActorSystem, cfg: Config, cfgPath: St
       toSequenceNr,
       Long.MaxValue,
       refreshInterval.orElse(Some(queryPluginConfig.refreshInterval)),
+      config.readProfile, // write journal read-profile
```

Comment:
this is used by multi dc persistence right? Maybe that should be passed in

Reply:
right, for the replication stream that is started from the ReplicatedEntity. It could work fine with a lower consistency for these queries, but it would not be evident that it's related to `journal-plugin.query.read-profile`. Not worth changing right now, but created ticket over there.